include/deal.II/grid/grid_tools.h

```diff
@@ -1375,7 +1377,9 @@ namespace GridTools
   template <int dim, int spacedim>
   void
   partition_triangulation (const unsigned int  n_partitions,
-                           Triangulation<dim, spacedim> &triangulation);
+                           Triangulation<dim, spacedim> &triangulation,
+                           const SparsityTools::Partitioner partitioner = SparsityTools::Partitioner::metis
```

Comment:
please adjust the documentation of this function (above) to reflect that there is now a choice between `METIS` and `Zoltan`.